public/scripts/form.js

```diff
@@ -90,7 +97,21 @@ function getData() {
     if (field.name === "debug" ) { data.debug = field.value; }
     if (field.name === "heroku_api_key" ) { data.herokuAPIKey = field.value.trim(); }
     if (field.name === "heroku_app_name" ) { data.herokuAppName = field.value.trim(); }
+    if (field.name === "subproject_url[]") {
```

Comment:
Is this a standard notation for representing arrays? If it is, then it's fine, otherwise remove it. `[]` in a field name looks weird.